README.mdx

```diff
@@ -5,13 +5,42 @@
 
 # Linea documentation
 
+## Quick Links 🔗
+- [Documentation Website](https://docs.linea.build/)
+- [Community Forum](https://community.linea.build/)
+- [Discord](https://discord.gg/linea)
+- [Create an Issue](https://github.com/Consensys/doc.linea/issues)
+- [Fork Repository](https://github.com/ConsenSys/doc.linea/fork)
+
+## What is Linea?
+
 [Linea](https://linea.build/) is a developer-ready layer 2 network, scaling Ethereum by providing
 an Ethereum-equivalent environment in which to execute transactions, which are then submitted to
 Ethereum Mainnet through a zero-knowledge rollup.
 
 This documentation repository is built using [Docusaurus](https://docusaurus.io/), and the site
 itself is published at [`docs.linea.build`](https://docs.linea.build/).
 
+## How to Contribute 🤝
```

Comment:
I'm not sure what this section adds. I appreciate that it condenses the main points into more digestible chunks, but at present it just seems to be duplicating information that we already have. What's the rationale here?